test/CMakeLists.txt

```
@@ -106,10 +116,21 @@ if(BUILD_PYTHON_INTERFACE)
 
   foreach(TEST_FILE ${${PROJECT_NAME}_PYTHON_UNITTEST})
     get_filename_component(TEST_NAME ${TEST_FILE} NAME_WE)
-    string(REGEX REPLACE "${PROJECT_SOURCE_DIR}/" "" TEST_FILE ${TEST_FILE})
+    string(REGEX REPLACE "${PROJECT_SOURCE_DIR}/" "" TEST_FILE_REL ${TEST_FILE})
```

No need of REGEX REPLACE here, REPLACE will work perfectly
```suggestion
    string(REPLACE "${PROJECT_SOURCE_DIR}/" "" TEST_FILE_REL ${TEST_FILE})
```